src/quo/components/profile/collectible_list_item/view.cljs

```diff
@@ -54,64 +84,83 @@
            {:type :communities
             :size :size-20}
            [avatar-image-src]]
-          [rn/view {:style {:width 8}}]
+
+          [reanimated/view
+           {:style (reanimated/apply-animations-to-style
```

Comment:
Under the hood, `reanimated/apply-animations-to-style` uses a hook.
https://github.com/status-im/status-mobile/blob/3c4f72c061bb697e3e0a973898f87275beeb2a98/src/react_native/reanimated.cljs#L120-L123
Here, we are calling it inside a condition, which is not ideal and can cause issues (see [Rules of Hooks](https://react.dev/reference/rules/rules-of-hooks)).

You are likely familiar with the hook rules, but the current function name doesn't indicate that it involves a hook.

I believe we decided to rename the function so that it starts with `use-`. cc: @Parveshdhull, @ulisesmac

Reply:
> and can cause issues

Probably can throw error like https://github.com/status-im/status-mobile/issues/16140

Reply:
logged an issue https://github.com/status-im/status-mobile/issues/20110

Reply:
Thank you @Parveshdhull

Reply:
Oh, I've seen @Parveshdhull already added the issue, Thank you!